content/source/docs/enterprise-beta/api/modules.html.md

```diff
@@ -0,0 +1,189 @@
+
+-> **Note**: These API endpoints are in beta and are subject to change.
+
+## List modules
+
+Lists all the modules for a given organization. **Note** that all other endpoints have the subpath `/api/v2`, this endpoint does not use that subpath.
+
+| Method | Path           |
+| :----- | :------------- |
+| POST | /api/registry/v1/modules/:organization |
+
+
+### Parameters
+- `:organization` (`string: <required>`) - Specifies the organization for which to list the modules
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $ATLAS_TOKEN" \
+  https://atlas.hashicorp.com/api/registry/v1/modules/skierkowski-v2
+```
+
+### Sample Response
+
+```json
+{
+  "meta": {
+    "limit": 15,
+    "current_offset": 0
+  },
+  "modules": [
+    {
+      "id": "skierkowski-v2/instance/aws/0.0.08",
+      "owner": "",
+      "namespace": "skierkowski-v2",
+      "name": "instance",
+      "version": "0.0.08",
+      "provider": "aws",
+      "description": "",
+      "source": "https://bitbucket-server.hashicorp.engineering/projects/SKI/repos/terraform-aws-instance/browse",
+      "published_at": "2017-11-30T07:00:40.664036Z",
+      "downloads": 0,
+      "verified": false
+    },
+    {
+      "id": "skierkowski-v2/vpc/aws/0.0.1",
+      "owner": "",
+      "namespace": "skierkowski-v2",
+      "name": "vpc",
+      "version": "0.0.1",
+      "provider": "aws",
+      "description": "",
+      "source": "https://bitbucket-server.hashicorp.engineering/projects/SKI/repos/terraform-aws-vpc/browse",
+      "published_at": "2017-11-30T00:53:06.966753Z",
+      "downloads": 0,
+      "verified": false
+    },
+    {
+      "id": "skierkowski-v2/subnet/aws/0.0.1",
+      "owner": "",
+      "namespace": "skierkowski-v2",
+      "name": "subnet",
+      "version": "0.0.1",
+      "provider": "aws",
+      "description": "",
+      "source": "https://bitbucket-server.hashicorp.engineering/projects/SKI/repos/terraform-aws-subnet/browse",
+      "published_at": "2017-11-30T17:38:01.600527Z",
+      "downloads": 0,
+      "verified": false
+    }
+  ]
+}
+```
+
+## Publish a Module
+
+This endpoint can be used to publish a new module to the registry. The publishing process will ingress the configuration from the repository and parse the various fields (e.g. input variables, output variables, readme) from the repository.
+
+| Method | Path           |
+| :----- | :------------- |
+| POST | /registry-modules |
+
+### Parameters
+
+- `linkable-repo-id` (`string: <required>`) - Specifies the repository to be used to ingress the configuration, this is in the format `<PROJECT_KEY>/<REPO>`
+- `o-auth-token-id` (`string: <requires>`) - Specifies the VCS Connection (OAuth Conection + Token) to use as identified. This ID can be obtained from the [o-auth-tokens](./o-auth-tokens.html) endpoint.
+
+
+### Sample Payload
+They payload must include `ingress-trigger-attributes`; however, those fields **MUST** be set to the default values provided.
```

Comment:
> the default values provided

Do you mean the values in the example below? I guess so but not 100% sure, if so I suggest just adding "in the example below" to make it crystal clear.

Reply:
Ping @ryanuber if we can get these defaulted before publishing these docs so that we don't need to specify them at all.

Reply:
Just checked with Ryan and this should be ready soon so I'm going to remove `ingress-trigger-attributes` altogether.